itm/models/equipment.py

```diff
@@ -119,10 +119,6 @@ def _get_type(self):
         help="A site or place of which this device is deployed",
     )
 
-    brand = fields.Many2one(
-        "itm.equipment.brand", "Brand Name", help="The assets or device brand"
-    )
-
```

Comment:
Why did you decide to remove the device brand here? If you have a lot of devices it might be useful to list only devices of a specific manufacturer.

Reply:
Because it is duplicated, see below:
`    brand_id = fields.Many2one("itm.equipment.brand", "Brand", help="The assets or device brand")`
If you want to use a brand, you need to add it to the form tree id="itm_equipment_tree", because the brand_id is listed there.
I like the 'brand_id' instead of brand, which is more correct. We can perform migration:
`        cr.execute("""
            UPDATE itm_equipment SET brand_id = brand WHERE brand_id IS NULL
         """)
`